pkg/skaffold/build/tag/date_time.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tag
+
+import (
+	"fmt"
+
+	"time"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+)
+
+const tagTime = "2006-01-02_15-04-05.999_MST"
```

Comment:
is this a valid image tag? I don't think we do any validation in the code on whether or not the tags we produce are valid according to the image tag spec

Reply:
Should be.

Reply:
yeah, this is valid. here's a snippet from a skaffold output for the default format: 

````
...
Build complete in 13.545233439s
gcr.io/***/svc-issues -> gcr.io/***/svc-issues:2018-05-31_17-44-51.176_UTC
gcr.io/***/petclinic-skaffold -> gcr.io/***/petclinic-skaffold:2018-05-31_17-44-56.952_UTC
...
````

I was thinking about validation, but then the tool fails with a very clear message when tagging fails, for example for 
````
     dateTime:
       format: "2006-01-02_15:04:05"
````
The colons are not valid, so: 

````
Successfully tagged 345165a2228c4b2f255f5eabcdd5724e:latest
FATA[0000] build step: tagging image: Error parsing reference: "gcr.io/***/svc-issues:2018-05-31_18:03:00" is not a valid repository/tag: invalid reference format 
 ````

Reply:
I think as long as the default format is valid its fine